app/src/pages/common/membersPage/membersPage.jsx

```diff
@@ -15,20 +15,15 @@ import { showNotification, NOTIFICATION_TYPES } from 'controllers/notification';
 import { fetch } from 'common/utils';
 import { URLS } from 'common/urls';
 import { withFilter } from 'controllers/filter';
-import { activeProjectSelector } from 'controllers/user';
+import { projectIdSelector } from 'controllers/pages';
 import { PaginationToolbar } from 'components/main/paginationToolbar';
 import { withPagination } from 'controllers/pagination';
 import { MEMBERS_PAGE, MEMBERS_PAGE_EVENTS } from 'components/main/analytics/events';
-import { PageLayout, PageHeader, PageSection } from 'layouts/pageLayout';
 import { MembersPageToolbar } from './membersPageToolbar';
 import { MembersGrid } from './membersGrid';
-import { NoResultsForFilter } from '../common/noResultsForFilter';
+import { NoResultsForFilter } from '../../inside/common/noResultsForFilter';
```

Comment:
May be it would be better to use absolute path for this import?

Reply:
+

Reply:
Fixed